_includes/projects.html

```diff
@@ -1,5 +1,3 @@
-<div class="project-container">
-{% for project in site.projects %}
```

Comment:
- Moved the logic for choosing which articles to show to the layouts
- Also moved the containing `project-container` div to the layouts